RadarSDK/RadarAPIClient.m

```diff
@@ -90,22 +105,20 @@ - (void)getConfig {
                                 sleep:NO
                            logPayload:YES
                     completionHandler:^(RadarStatus status, NSDictionary *_Nullable res) {
-                        if (!res) {
-                            return;
-                        }
-        
-                        [Radar flushLogs];
+        [Radar flushLogs];
 
-                        id metaObj = res[@"meta"];
-                        if (metaObj && [metaObj isKindOfClass:[NSDictionary class]]) {
-                            NSDictionary *meta = (NSDictionary *)metaObj;
-                            id configObj = meta[@"config"];
-                            if (configObj && [configObj isKindOfClass:[NSDictionary class]]) {
-                                NSDictionary *config = (NSDictionary *)configObj;
-                                [RadarSettings setConfig:config];
-                            }
-                        }
-                    }];
+        if (!res) {
+            return;
+        }
+        
+        RadarMeta *meta = [RadarAPIClient parseMeta:res];
+        if (meta.config) {
+            [RadarSettings setTrackingOptions:meta.trackingOptions];
```

Comment:
@jkao looks like this is inconsistent with your [Android changes](https://github.com/radarlabs/radar-sdk-android/pull/135/files#diff-a114947704d76fe205df21d9d50315f63e0368c5d164a206b3ccaeffa521ca75R115) Can you confirm expected behavior? The Android changes return to the callback, which calls the `updateTrackingFromMeta` method ([which in turns calls this method](https://github.com/radarlabs/radar-sdk-android/pull/135/files#diff-80113b4a9caa6ded13eca3973127e13b9053f26e6c32bb2f1b279809f5c446caR256)).

Reply:
yes, this looks inconsistent. probably a merge conflict. let me check how the callback is handled, but I believe we can remove this `if` block entirely.

Reply:
with the downstream caller handling it:

```
        RadarMeta *meta = [RadarAPIClient parseMeta:res];
        completionHandler(status, meta);
```

Reply:
good catch!

Reply:
So I should just remove the `if` block, and that's all?

Reply:
@jrtibbetts yes.

Reply:
Done.